wordpress-mobile/WordPress-Android
# Switch the update_ps_strings lane to the shared action

This PR moves the `update_metadata_source` fastlane action to the shared plugin in order to reuse it in the Woo app.

To test:
1. Run `bundle exec fastlane update_ps_strings version:11.6` and verify that nothing changes in the .po file.
2. Make some changes to one of the .txt files and commit them.
3. Run `bundle exec fastlane update_ps_strings version:11.6` and verify that the .po file has been correctly updated. 

### Note: 
This PR currently references a commit hash. As soon as this is approved, I'll release a new version of `release-toolkit` and I'll update this PR to reference it.

Update release notes:
